client/hotrod-client/src/test/java/org/infinispan/client/hotrod/HitsAwareCacheManagersTest.java

```diff
@@ -200,23 +201,27 @@ protected void destroy() {
    public static class HitCountInterceptor extends CommandInterceptor{
       private static final Log log = LogFactory.getLog(HitCountInterceptor.class);
 
-      private volatile int invocationCount;
+      private final AtomicInteger invocationCount = new AtomicInteger(0);
 
       @Override
       protected Object handleDefault(InvocationContext ctx, VisitableCommand command) throws Throwable {
          if (ctx.isOriginLocal()) {
-            log.infof("Increment hit count after being visited by %s", command);
-            invocationCount ++;
+            if (Thread.currentThread().getName().contains("NodeB")) {
```

Comment:
@wburns Sebastian already integrated it, but I'm opening a follow-up PR to remove this and convert the interceptor to `BaseAsyncInterceptor`.